### DISCUSSION TITLE
欢迎来到「现代 JavaScript 教程」读者讨论区！

### DISCUSSION THREAD

**Discussion by leviding** [javascript-tutorial/zh.javascript.info#860] [General]
## 👋 欢迎！
  这里是「现代 JavaScript 教程」读者讨论区，所有人都可以在这讨论和提问。我们希望你：
  * 提出你想知道的问题。
  * 分享有趣的想法。
  * 与社区中其他成员互动。
  * 欢迎他人，思想开放。请记住这是我们一起建立的社区 💪。

  现在，你就可以在下方做一下自我介绍，并告诉我们你对「现代 JavaScript 教程」的看法啦 👀。

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

| 微信扫码关注官方订阅号，订阅更多精彩内容 |
|-|
| <img src="https://user-images.githubusercontent.com/26959437/120764347-a3b91680-c54a-11eb-9a15-31bb2596b0db.png" width="500px;" /> |
| <img src="https://user-images.githubusercontent.com/26959437/120764353-a61b7080-c54a-11eb-8a79-75177c342b45.png" width="500px;" /> |

**加入读者交流群：**

- **微信群**：加微信 **`imleviding`** 或 [扫二维码](https://user-images.githubusercontent.com/26959437/64596211-0fa63380-d3e6-11e9-983d-bd788ffe2370.jpg)，验证信息填写 **`JS 教程`**。
- **QQ 群**：打开 QQ 搜索群号 **`955916282`** 或 [扫二维码](https://user-images.githubusercontent.com/26959437/138594800-29700a46-0cb1-45bd-b614-4b6486cf0124.jpg)，验证信息填写 **`JS 教程`**。
- **Telegram**：https://t.me/TheModernJavaScriptTutorial
- **Gitter**：[点击加入 Gitter Room](https://gitter.im/zh-javascript-info/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

**教程配套每日一题**：https://github.com/leviding/one-note-a-day

**Comment by xujunjiong** [comment]
我要当第一个评论的

**Comment by QuentinHsu** [comment]
围观大哥！

**Comment by xiaozhi1218** [comment]
围观

**Comment by moemoechu** [comment]
哇哦，赞一个。

**Comment by CatSayInk** [comment]
第二个！

**Comment by FHSWar** [comment]
乌拉!

**Comment by l6y8n9x** [comment]
反手就是一个赞

**Reply by wuzhidexiaoming** [reply]
阔以阔以

**Comment by wuzhidexiaoming** [comment]
阔以的

**Comment by jiang46633016** [comment]
Good like

**Comment by FN400** [comment]
写的非常仔细，赞

**Comment by CZT0** [comment]
有没有 现代 TypeScript 简明教程 🤭

**Comment by WMCwmc503** [comment]
箭头函数中 “动态创建一个函数” 这个例子的第4行代码编写有个小失误，不知道在哪里提交……

**Reply by leviding** [reply]
> 

提 Pull Request

**Comment by riohuky** [comment]
有没有关于ES6的文章？
